### PR TITLE
Update libraries, remove apache commons-io, code clean up

### PR DESCRIPTION
**Pull request by thakkaryash94**
Apache common io is very heavy library and we are using only two methods.
1. closequietly is deprecated in 2.6(https://github.com/apache/commons-io/blob/master/src/main/java/org/apache/commons/io/IOUtils.java#L58). So manually try-catch it with ignored exception.
2. copy inputstream to outputstream. This is very critical and important method for us. So directly copy it from apache source and put it under utils with same class(IOUtils).
Remove unused svg

### This is very critical PR. I attempt to clean up few things. It may require more changes. Please go through changes and verify it personally.

I personally test sample app with all examples on real device but highly recommend that you personally review it.

### CODE REVIEW

File: FrescoImageLoader/src/main/java/com/github/piasy/biv/loader/fresco/ImageDownloadSubscriber.java [hunk 1]
```diff
@@ -77,8 +77,14 @@ protected void onNewResultImpl(DataSource<CloseableReference<PooledByteBuffer>>
         } catch (IOException e) {
             onFail(e);
         } finally {
-            IOUtils.closeQuietly(inputStream);
-            IOUtils.closeQuietly(outputStream);
+            try {
```
Comment (Piasy) [inline FrescoImageLoader/src/main/java/com/github/piasy/biv/loader/fresco/ImageDownloadSubscriber.java]: Why not keep `closeQuietly` in the copied `IOUtils` class?

Comment (thakkaryash94) [reply FrescoImageLoader/src/main/java/com/github/piasy/biv/loader/fresco/ImageDownloadSubscriber.java]: Means our own try catch method with closeQuietly name??

**Comment by Piasy** [comment]
Thanks for your contribution! I'll cut a release soon.

**Comment by Piasy** [comment]
`v1.4.6` is released, check it out!

**Pull request merged**